airflow/settings.py

```diff
@@ -148,18 +148,6 @@ def configure_orm(disable_connection_pool=False):
         # 0 means no limit, which could lead to exceeding the Database connection limit.
         pool_size = conf.getint('core', 'SQL_ALCHEMY_POOL_SIZE', fallback=5)
```

Comment:
Shouldn't we remove this too?

Reply:
https://docs.sqlalchemy.org/en/13/core/pooling.html#sqlalchemy.pool.SingletonThreadPool

`SingletonThreadPool` has `pool_size` parameter.